tests/checks/__snapshots__/PrismaClientOnContext.test.ts.snap

```diff
@@ -68,13 +68,7 @@ exports[`duckType strategy: passes if client looks like a valid prisma client in
 
 exports[`instanceOf strategy: If prisma client import fails then the check cannot perform its instanceof strategy and thus throws an error: graphqlOperationExecutionResult 1`] = `
 Object {
-  "data": Object {
-    "users": Array [
-      Object {
-        "profile": null,
-      },
-    ],
-  },
+  "data": null,
```

Comment:
Why did this snap change here?

Reply:
By mistake I will fix this

Reply:
This snap-change because the prisma schema changed and profile now is required